microbench/src/test/java/io/netty/microbench/buffer/PooledByteBufAllocatorBenchmark.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2015 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package io.netty.microbench.buffer;
+
+import io.netty.buffer.ByteBuf;
+import io.netty.buffer.ByteBufAllocator;
+import io.netty.buffer.PooledByteBufAllocator;
+import io.netty.microbench.util.AbstractMicrobenchmark;
+import org.openjdk.jmh.annotations.Benchmark;
+import org.openjdk.jmh.annotations.Level;
+import org.openjdk.jmh.annotations.Setup;
+import org.openjdk.jmh.annotations.Threads;
+
+@Threads(1)
+public class PooledByteBufAllocatorBenchmark extends AbstractMicrobenchmark {
+
+    private static final ByteBufAllocator ALLOC = PooledByteBufAllocator.DEFAULT;
+    private static final int SIZE = 1024;
+    private static final int ALLOCATIONS = 50;
+    private static final byte BYTE = '0';
+
+    private final ByteBuf[] buffers = new ByteBuf[256];
```

Comment:
nope...we need to have more buffers in the local-cache then the number we need to allocate as otherwise LIFO and FIFO will behave the same way. That said I just noticed that we try to put more buffers in the cache as it can hold (as it is limited to 64 buffers for this size). Let me increase the number and see.